ui/src/app/ErrorInfo/ErrorInfo.tsx

```diff
@@ -1,38 +1,41 @@
 import React from 'react';
 import { Popover, Icon, Tooltip } from '@patternfly/react-core';
-import { TableComposable, Tr, Th, Td } from '@patternfly/react-table'
+import { TableComposable, Tr, Th, Td, Tbody } from '@patternfly/react-table';
 import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
 import { DeploymentStepStatusData } from '@app/apis/types';
 
-import './ErrorInfo.css'
+import './ErrorInfo.css';
 
 interface IErrorInfoProps {
   stepStatusData: DeploymentStepStatusData;
 }
 
 export const ErrorInfoPopover = ({ stepStatusData }: IErrorInfoProps) => {
-
   const data = (
     <>
-    <TableComposable>
-        <Tr>
+      <TableComposable>
+        <Tbody>
+          <Tr>
             <Th>Error:</Th>
-            <Td className='errorCell'>{stepStatusData.error}</Td>
-        </Tr>
-        <Tr>
+            <Td className="errorCell">{stepStatusData.error}</Td>
+          </Tr>
+          <Tr>
             <Th>Details:</Th>
-            <Td className='errorDetailCell'><div className='errorDetailDiv'>{stepStatusData.errorDetails}</div></Td>
-        </Tr>
-        <Tr>
-            <Th modifier='nowrap'>Correlation ID:</Th>
+            <Td className="errorDetailCell">
+              <div className="errorDetailDiv">{stepStatusData.errorDetails}</div>
+            </Td>
+          </Tr>
+          <Tr>
+            <Th modifier="nowrap">Correlation ID:</Th>
             <Td>{stepStatusData.correlationId}</Td>
-        </Tr>
-    </TableComposable>
+          </Tr>
+        </Tbody>
+      </TableComposable>
     </>
   );
   return (
-    <Tooltip content={<div>Click for error info</div>}>
-      <Popover minWidth="40rem" bodyContent={<div>{data}</div>}>
+    <Tooltip removeFindDomNode={true} content={<div>Click for error info</div>}>
+      <Popover removeFindDomNode={true} minWidth="40rem" bodyContent={<div>{data}</div>}>
```

Comment:
No need to wrap the `data` in a `<div> </div>` it already has only one top level element.

Reply:
OK!